biggus/tests/test_linear_mosaic.py

```diff
@@ -219,6 +219,20 @@ def test_ndarray(self):
                   [8, 9, 10, 11, 10, 11, 12, 13, 14]]
         np.testing.assert_array_equal(result, target)
 
+    def test__getitem_full_keys(self):
+        full_keys = (np.array([0, 1, 2, 3]), slice(None, None, None),
+                     slice(None, None, None), slice(None, None, None))
+        tile1 = self._tile((4, 4, 5, 11))
```

Comment:
Would you mind reducing this test to 2 tiles, and 2 keys? I think it makes the test look more complex than it is.